Engine/src/mini/input/Keyboard.java

```diff
@@ -42,4 +71,13 @@ public static boolean isKeyUp(int key) {
         Integer action = keys.get(key);
         return action != null && (action.equals(GLFW_RELEASE));
     }
+
+    private static KeyboardKey getKey(int key) {
+        for(KeyboardKey k : KeyboardKey.values()) {
```

Comment:
Formatting for (KeyboardKey k : KeyboardKey.values()), mellemrum mellem for :dagger: